scriptworker/config.py

```diff
@@ -65,24 +65,27 @@ def unfreeze_values(dictionary):
     A recursive function(top-down conversion)
```

Comment:
I think making both functions pure is a good goal. I do want to avoid a complete rewrite of both functions atm, if possible, since a contributor just wrote these.